sddi-base/Dockerfile

```diff
@@ -192,6 +192,7 @@ RUN set -ex && \
   ckan config-tool "${CKAN_INI}" "scheming.dataset_schemas = ckanext.scheming:ckan_dataset.yaml" && \
   ckan config-tool "${CKAN_INI}" "scheming.presets = ckanext.scheming:presets.json ckanext.repeating:presets.json ckanext.composite:presets.json" && \
   ckan config-tool "${CKAN_INI}" "scheming.dataset_fallback = false" && \
+  ckan config-tool "${CKAN_INI}" "licenses_group_url = https://raw.githubusercontent.com/tum-gis/ckanext-grouphierarchy-sddi/main/ckanext/grouphierarchy/licenses_SDDI.json" &&\
```

Comment:
- Missing space after `&&`
- Is setting this here really required? If this file is the default, it should be shipped with the extension (which is the case, according to the URL) and set as the default value (local path inside the container) in the extension. The user should only have to take action, if he needs to overwrite this.

Reply:
This is an extended license list  which is required for SDDI. With CKAN installation it comes [default licenses list](https://docs.ckan.org/en/2.9/maintaining/configuration.html#licenses-group-url).
With the URL which is defined in mentioned [Dockerfile](https://github.com/tum-gis/ckan-docker/blob/devel-update/sddi-base/Dockerfile#L195), I want to overwrite CKAN Default value and use customized list.

This value could be defined as ENV var which I've added in our [docker/compose](https://github.com/tum-gis/ckan-docker/blob/feature/compose/compose/2.9/ckan.env#L23-L24) installation draft.